api/base.py

```diff
@@ -255,6 +255,11 @@ def handle_exception(self, exception, debug):
             code = 400
         else:
             code = 500
+
+        if code == 500:
```

Comment:
This could also be a check for 5xx errors - we return 503s for DB or Elastic unavailability. Not sure we would need a stacktrace for that.